tools/packaging/packaging.psm1

```diff
@@ -3090,6 +3075,77 @@ function New-MSIPackage
     }
 }
 
+function New-MsiArgsArray {
+    param(
+        [Hashtable]$Argument
+    )
+
+    $buildArguments = @()
+    foreach ($key in $Argument.Keys) {
+        $buildArguments += "-d$key=`"$($Argument.$key)`""
+    }
+
+    return $buildArguments
+}
+
+function Start-MsiBuild {
+    param(
+        [string[]] $WxsFile,
+        [string[]] $Extension = @('WixUIExtension', 'WixUtilExtension'),
```

Comment:
If you're gearing up to create a bundle, you may as well as the WixBalExtension here as well. Even if you don't use it now, it won't cause any harm (like linking a .obj file in VC++ you don't use).

Reply:
```suggestion
        [string[]] $Extension = @('WixUIExtension', 'WixUtilExtension', 'WixBalExtension'),
```